src/plugins/focus.service.ts

```diff
@@ -0,0 +1,41 @@
+import {Injectable, Optional} from '@angular/core';
+import { DisposableView } from 'ng2-qgrid/core/view/disposable.view';
+import { RootService } from 'ng2-qgrid/infrastructure/component/root.service';
+
+@Injectable()
+export class FocusService extends DisposableView {
+	constructor(@Optional() private root: RootService) {
+		super();
+
+		this.model = root.model;
+	}
+
+	private model;
+
+	private focus() {
+		this.root.table.view.focus();
+	}
+
+	activateAfterRender(state: string, ...args) {
```

Comment:
More common to place public methods before private. It allows to find methods which can be used to serve programmer purposes easier.

Reply:
ok)